### DISCUSSION TITLE
How to contribute to docs

### DISCUSSION THREAD

**Discussion by atareversei** [basliqlabs/qwest-docs#15] [Announcements]
1. first clone the repo
2. create a new branch with the following format: `github-username/[docs|blog]/document-name`
3. use semantic commit messages: `git commit -m "[docs|blog](feature): description"`
4. `npm i` to install the packages
5. `npm run start` to check the updates you have made
6. create a pull request if you are done

---

### Docusaurus gives 404 error when `md` file is present
Please put a heading one `# Title` at the top of your file to let Docusaurus know that you have created an entry to that URL.

**Comment by thisis-mahyar** [comment]
@atareversei
- I've been creating a branch for each doc file.
- (For more clearity) This is an example of the format I've used for commiting: [doc] (update) add h2

**Reply by atareversei** [reply]
By doc file, you mean each feature? e.g. trello and github-projects? If this is the case, that might be ok.
...and why are you using a different format than what I've specified? Your commit message should look like: `docs(trello): add h2` and please don't use any other message format.

**Reply by thisis-mahyar** [reply]
Yes exactly 👌
Understood! I had doubts about the format. All right then 👍
